WcaOnRails/db/migrate/20180107142301_update_feet_format_in_rounds.rb

```diff
@@ -0,0 +1,10 @@
+class UpdateFeetFormatInRounds < ActiveRecord::Migration[5.1]
+  def change
+    competitions_with_feet = Competition.includes(competition_events:[:rounds]).has_event("333ft").where("end_date >= ?", Date.new(2018, 1, 1))
+    rounds_with_feet_mo3 = competitions_with_feet.map(&:competition_events).flatten.select { |e| e.event_id == "333ft" }.map(&:rounds).flatten.select { |r| r.format_id == "m" }
+    rounds_with_feet_mo3.each do |r|
+      r.format_id = "a"
+      r.save
```

Comment:
We are usually more strict in migrations, so I'd suggest `r.save!` or more concisely `r.update! format_id: "a"`.